src/components/Button.tsx

```diff
@@ -0,0 +1,27 @@
+import cn from 'classnames';
+import { MouseEventHandler, ReactNode } from 'react';
+
+type Button = {
+  type: "button" | "submit" | "reset" | undefined;
+  className: string;
```

Comment:
Can we make this optional? Using:

```suggestion
  className?: string;
```